heron/mlmgr/src/main/java/org/apache/heron/LocalStormDoTask.java

```diff
@@ -0,0 +1,82 @@
+package org.apache.heron;
```

Comment:
the package path should be `org.apache.heron.ml` if it's a general class needed by all machine learning runtime. If it's specific for samoa, then the package path should be `org.apache.heron.ml.samoa`. 

And this also applies to all the following files.